packages/metacoin/test/utils/web3_wrapper.ts

```diff
@@ -0,0 +1,30 @@
+import { env, EnvVars } from '@0xproject/dev-utils';
+import { GanacheSubprovider } from '@0xproject/subproviders';
+import { Web3Wrapper } from '@0xproject/web3-wrapper';
+import * as fs from 'fs';
+import * as _ from 'lodash';
+import ProviderEngine = require('web3-provider-engine');
+
+import { coverage } from './coverage';
+
+export const web3Provider = new ProviderEngine();
+const isCoverageEnabled = env.parseBoolean(EnvVars.SolidityCoverage);
+if (isCoverageEnabled) {
+    web3Provider.addProvider(coverage.getCoverageSubproviderSingleton());
+}
+web3Provider.addProvider(
+    new GanacheSubprovider({
+        logger: {
+            log: (arg: any) => {
+                fs.appendFileSync('ganache.log', `${arg}\n`);
+            },
+        },
+        verbose: env.parseBoolean(EnvVars.SolidityCoverage),
+        port: 8545,
+        networkId: 50,
+        mnemonic: 'concert load couple harbor equip island argue ramp clarify fence smart topic',
```

Comment:
It would be great if we have a `config.ts` file with all the configurable aspects of the Metacoin project. This is really easy for people to edit and to see all that can be easily configured in one place.